packages/react-ui/src/widgets/CategoryWidgetUI/CategoryWidgetUI.js

```diff
@@ -476,7 +502,9 @@ CategoryWidgetUI.propTypes = {
   animation: PropTypes.bool,
   filterable: PropTypes.bool,
   searchable: PropTypes.bool,
-  isLoading: PropTypes.bool
+  isLoading: PropTypes.bool,
+  localizationMessages: PropTypes.object,
```

Comment:
I wouldn't recommend this approach... Is this a must for you guys?

Reply:
It's a first approach. We can discuss